hugegraph-api/src/main/java/com/baidu/hugegraph/core/GraphManager.java

```diff
@@ -157,6 +159,10 @@ public HugeGraph cloneGraph(String name, String newName,
     }
 
     public HugeGraph createGraph(String name, String configText) {
+        E.checkArgument(hugeConfig.get(ServerOptions.ENABLE_DYNAMIC_CREATE_DROP),
+                        "Not allowed to create graph dynamically, " +
+                                "please set `enable_dynamic_create_drop` to true.",
+                        name);
```

Comment:
need to fill graph name parameter: "Not allowed to create graph dynamically: '%s', "?